### PR TITLE
Updating usecase

### PR DESCRIPTION
**Pull request by stokljos**
changed a section and updated some files.

### CODE REVIEW

File: docs/lowes/usecase.md [hunk 1]
```diff
@@ -1,89 +1,190 @@
...
-```math
-f(x) = \int_{-\infty}^\infty
-\hat f(\xi)\,e^{2 \pi i \xi x}
-\,d\xi
-```
+  
+## Running Opensees through HPC
 
-### Code
+(Script needs to be established on design safe. I have a working notebook, just need to connect it with modeling script)
```
Comment (kks32) [inline docs/lowes/usecase.md]: I'll wait until you complete this...

**Review by kks32 — CHANGES_REQUESTED** [review]
I'd request to move the description of different functions from the usecase file to ipynb files in DesignSafe. Could you please add descriptions and comments to the ipynb files?

The usecase can give a high-level overview of the different sections.

**Comment by stokljos** [comment]
Hello,

I have updated notebooks I am trying to add to design safe but I am unable
to. What's the best way to do this for the use case jupyter files? All
those files have descriptions.

And then following what you are saying, I shouldn't a quick summary of the
outputs for each notebook? I thought that would give the user a good
overview of how the notebook works and what is expected for the output?

I am also no longer a student and have a pretty full schedule at work. I
have not been to a designsafe meeting in almost a year as well so i am not
update to date. I was contacted by my professor just to update the
notebooks, I have very limited time before the 15th as I am in the middle
of moving and on vacation this next week.

**Comment by kks32** [comment]
@stokljos I have merge conflicts. Could you please resolve those?

**Review by kks32 — APPROVED** [review]

**Pull request merged**